### ISSUE TITLE
Prefer to export complex view logic to POROs and Decorators instead of global accessible view helper module

### ISSUE THREAD

**Issue by peterhassler** [simpliflow/Rails-Styleguide#3]
https://github.com/simpliflow/Rails-Styleguide#no-complex-view-formatting

Moving complex formatting from views into view helpers is one way to tackle fat views. 
In bigger applications like our one, there are several problems with that:
- more view partials need more complex logic in view helpers
- many parts of this extracted view logic don't get reused anywhere but in the view it was extracted from
  - since view helpers are 'global' to every view, we lose track of where these helper methods belong
  - the logic may be so special that naming them correctly is hard (possible name collision)
  - implicit dependencies through 'global' space

To tackle these problems, complex view formatting logic should be extracted into (in this order) ...
1. POROs: Plain Old Ruby Objects (+ Explicit dependencies, + Naming describes context, + Easily testable, - Need to manually include other view helpers)
2. Decorators (~ Explicit dependencies, + Naming describes context, ~ Better Testable, + View Helper access already setup)
3. (for generic cases) View Helpers (- Explicit dependencies, - Naming, - Testable, + View helpers accessible)

**Comment by tomgong** [comment]
I agree with most of the points although I think that decorators are preferable in case of resource and context specific view helpers. But keep in mind to decorate at the end of the action to avoid using decorator functionality within business logic.
